IBM_DB/ibm_db/README

```diff
@@ -1,98 +1,42 @@
-Python Driver and DBI Wrapper for IBM Databases
------------------------------------------------
-The ibm_db driver and ibm_db_dbi wrapper provide a Python interface for IBM 
-databases.  The wrapper conforms to the DBI specification located here:
-  http://www.python.org/dev/peps/pep-0249/
+= Installing ibm_db and ibm_db_dbi module =
+
+We are assuming that you have Python already installed. In Linux you may need the python-dev package (you can install python-dev package through "$yum install python-devel" if yum doesn't work then you can also install it through "$apt-get install python-dev")
+
+ 
+== Installation ==
+{{{
+  pip install ibm_db
+}}}
+ 
+This will install *ibm_db* and *ibm_db_dbi* module.
+
+ == IBM_DB and DB-API wrapper (ibm_db_dbi) sanity test ==
+
+{{{
+$ python
+Python 2.5.1 (r251:54863, Oct  5 2007, 13:36:32)
+[GCC 4.1.3 20070929 (prerelease) (Ubuntu 4.1.2-16ubuntu2)] on linux2
+Type "help", "copyright", "credits" or "license" for more information.
+>>> import ibm_db
+>>> ibm_db_conn = ibm_db.connect('pydev', 'db2inst1', 'secret')
+>>> import ibm_db_dbi
+>>> conn = ibm_db_dbi.Connection(ibm_db_conn)
+>>> conn.tables('SYSCAT', '%')
+}}}
+
+==Supported databases==
+ * *Minimum Supported version of IBM DB2 is V9fp2 for Linux, UNIX, and Windows
+ * *Informix 11.10* (Cheetah)
+ * *Remote connections to i5/OS (iSeries)*
+ * *Remote connections to z/OS (DB2 for z/OS)*
+
+==Feedback==
+
+Your feedback is very much appreciated and expected through project ibm-db:
+  * *ibm-db issues reports*:  https://github.com/ibmdb/python-ibmdb/issues
+  * *ibm_db discuss*: http://groups.google.com/group/ibm_db
+  * *ibm-db developers*:      opendev@us.ibm.com
 
-The driver and wrapper have been developed and will be supported by IBM. 
```

Comment:
please pull in the contributions page from https://code.google.com/p/ibm-db/wiki/Contributions (repo names will change). Build instructions could probably go there.